plugins/octant/cmd/antrea-octant-plugin/traceflow.go

```diff
@@ -171,40 +190,68 @@ func (p *antreaOctantPlugin) actionHandler(request *service.ActionRequest) error
 			if s == nil {
 				log.Printf("Invalid user input, CRD creation or Traceflow request may fail: "+
 					"failed to get destination IP as a valid IPv4 IP: %s", err)
+				alert := action.CreateAlert(action.AlertTypeError, fmt.Sprintf("Invalid destination IPv4 string, "+
+					"please check your input and submit again."), action.DefaultAlertExpiration)
+				request.DashboardClient.SendAlert(request.Context(), request.ClientID, alert)
+				return nil
 			}
 			if s.To4() == nil {
 				log.Printf("Invalid user input, CRD creation or Traceflow request may fail: "+
 					"failed to get destination IP as a valid IPv4 IP: %s", err)
+				alert := action.CreateAlert(action.AlertTypeError, fmt.Sprintf("Invalid destination IPv4 string, "+
+					"please check your input and submit again."), action.DefaultAlertExpiration)
+				request.DashboardClient.SendAlert(request.Context(), request.ClientID, alert)
+				return nil
 			}
 			destination = opsv1alpha1.Destination{
 				IP: dst,
 			}
 		case opsv1alpha1.DstTypeService:
-			if match := regExpMatch(namespaceStrPattern, dstNamespace); !match {
+			if errs := validation.ValidateNamespaceName(dstNamespace, false); len(errs) != 0 {
+				log.Printf("Invalid user input, CRD creation or Traceflow request may fail: "+
+					"failed to validate destination namespace string %s, errs: %#v", dstNamespace, errs)
+				alert := action.CreateAlert(action.AlertTypeError, fmt.Sprintf("Invalid destination namespace string, "+
+					"please check your input and submit again."), action.DefaultAlertExpiration)
+				request.DashboardClient.SendAlert(request.Context(), request.ClientID, alert)
+				return nil
+			}
+			if errs := validation.NameIsDNS1035Label(dst, false); len(errs) != 0 {
 				log.Printf("Invalid user input, CRD creation or Traceflow request may fail: "+
-					"failed to match namespace %s with K8s Namespace pattern %s", dstNamespace, namespaceStrPattern)
+					"failed to validate destination service string %s, errs: %#v", dst, errs)
+				alert := action.CreateAlert(action.AlertTypeError, fmt.Sprintf("Invalid destination service string, "+
+					"please check your input and submit again."), action.DefaultAlertExpiration)
+				request.DashboardClient.SendAlert(request.Context(), request.ClientID, alert)
+				return nil
 			}
 			destination = opsv1alpha1.Destination{
 				Namespace: dstNamespace,
 				Service:   dst,
 			}
 		}
 
+		// It is not required for users to input port numbers.
```

Comment:
Then the following logging on L236 and L242 is misleading?

Reply:
Deleted these logs.